src/ui/MLFacet/MLFacet.ts

```diff
@@ -329,6 +329,16 @@ export class MLFacet extends Component {
     this.updateAppearance();
   }
 
+  /**
+   * Tells the API if the sent facet values should be returned in the same order or not.
```

Comment:
Suggestion:

> Sets a flag indicating whether the facet values should be returned in their current order.
>
> Setting the flag to `true` helps ensuring that the facet does not move around while the end-user is interacting with its values.
>
> The flag is automatically set back to `false` after a query is built.

Reply:
Thanks! Really wasnt sure about that one!

Reply:
Hmm I might change the second line for

> Setting the flag to true helps ensuring that the facet values do not move around while the end-user is interacting them.

Because this only ensures the values don't move for now

Reply:
Sure :)